### PR TITLE
Init container capabilities in the pod

### PR DESCRIPTION
**Pull request by knabben**
* Adding initContainer capability in the pod
* Renaming folder `data` to `entities`
* Fixes https://github.com/K8sbykeshed/k8s-service-lb-validator/issues/44

### CODE REVIEW

File: main_test.go [hunk 1]
```diff
@@ -20,42 +21,60 @@ import (
...
 	ctx       = context.Background()
 	logger    *zap.Logger
 )
 
+
 func init() {
 	var err error
-	if logger, err = zap.NewProduction(); err != nil {
+	if logger = NewLoggerConfig(); err != nil {
 		log.Fatal(err)
```
Comment (yzaccc) [inline main_test.go:37]: looks like no need to check err here, as no err is returned.

Comment (knabben) [reply main_test.go:37]: we should fix this logger thing. Suggestion of another library, or improvement in the setup, the worst thing is the need to pass the logger down the stack manager/models etc

File: matrix/helper.go [hunk 2]
```diff
@@ -57,7 +56,7 @@ func ValidateOrFail(k8s *KubeManager, model *Model, testCase *TestCase, ignoreLo
 	testCase.Reachability.PrintSummary(true, true, true)
 
 	if wrong == 0 {
-		k8s.Logger.Info("VALIDATION SUCCESSFUL")
+		k8s.Logger.Info("Test passes, validation successful!")
```
Comment (yzaccc) [inline matrix/helper.go:59]: nit： Tests passed, validation succeeded!

File: matrix/manager.go [hunk 3]
```diff
@@ -3,6 +3,7 @@ package matrix
 import (
 	"context"
 	"fmt"
+	"github.com/k8sbykeshed/k8s-service-lb-validator/entities"
```
Comment (yzaccc) [inline matrix/manager.go:6]: goimports -> group the packages, move this line to 19

File: matrix/manager.go [hunk 4]
```diff
@@ -72,7 +72,7 @@ func (k *KubeManager) InitializeCluster(model *Model, nodes []*v1.Node) error {
 		if err != nil {
 			return err
 		}
-		k.Logger.Info("Wait for pod running.", zap.String("name", kubePod.Name), zap.String("namespace", kubePod.Namespace))
+		k.Logger.Debug("Wait for pod running.", zap.String("name", kubePod.Name), zap.String("namespace", kubePod.Namespace))
 		if err := k8s.WaitForPodRunningInNamespace(k.clientSet, createdPod); err != nil {
```
Comment (yzaccc) [inline matrix/manager.go:76]: createdPod -> kubePod

**Review by yzaccc — APPROVED** [review]
LGTM

**Pull request merged**